openmcyclus/DepleteReactor.py

```diff
@@ -609,63 +627,29 @@ def transmute(self):
         '''
         assemblies = self.core.pop_n(self.core.count)
         self.core.push_many(assemblies)
-        ss = str(len(assemblies)) + " assemblies"
+        # ss = str(len(assemblies)) + " assemblies"
         # self.record("TRANSMUTE", ss)
```

Comment:
Consider removing these if they are not going to be used.

Reply:
Any instances of the record method are intended to be there, but I need to dig into the cyclus python API for writing to the database some to make sure it all works as it should/does in the c++ api

Reply:
Can you explain the purpose of having this method present in the code but commented out? Is there still something you are debugging? Or does it need to be there commented out to work properly?

Reply:
The `self.record` method writes information to the database, and in this case trying to record when different reactor-related events occur. When I was working on this archetype, I noticed that it this method wasn't working as expected, likely because of how the Cyclus python API implements it. So I commented it out to prevent it from hindering other development work on the archetype. I didn't remove it, because I do want this functionality in the code, but it's not crucial. I will make an issue about getting this working to make sure it doesn't get forgotten.